js/views/chatHeadsVw.js

```diff
@@ -18,6 +18,18 @@ module.exports = baseVw.extend({
         this.createChatHead(md).render().el
       );
     });
+
+    this.$chatHeadsContainer = $('.chatConversationHeads');
```

Comment:
Can you pass this element into the view (maybe as $scrollContainer)? It's cached in the parent views as this.$chatHeadsContainer. It's minor, but I think that's a better practice than targeting it globally.